assets/javascripts/discourse/components/subscribe-country-select.js

```diff
@@ -0,0 +1,274 @@
+import ComboBoxComponent from "select-kit/components/combo-box";
+import { computed } from "@ember/object";
+import I18n from "I18n";
+
+export default ComboBoxComponent.extend({
+  pluginApiIdentifiers: ["subscribe-country-select"],
+  classNames: ["subscribe-address-country-select"],
+  nameProperty: "name",
+  valueProperty: "value",
+
+  selectKitOptions: {
+    filterable: true,
+    allowAny: false,
+    translatedNone: I18n.t(
+      "discourse_subscriptions.subscribe.cardholder_address.country"
+    ),
+  },
+
+  content: computed(function () {
```

Comment:
Minor, would be good to have the country names be translatable. 

(I would have thought we'd have a list of countries somewhere... but it doesn't look like it.)

Reply:
To be clear, this can be in a follow-up.

Reply:
I wondered if we had a list somewhere else too. I lifted all of these from hosted-site, expanded the states list to include territories, and renamed them to avoid collisions. I'll make a follow-up note for this one. I've got another enhancement in mind for this plugin.